### DISCUSSION TITLE
Few questions about BLE and USB

### DISCUSSION THREAD

**Discussion by c0debreaker** [gopro/OpenGoPro#356] [Q&A]
I tried OpenGopro yesterday on my linux using bluetooth dongle and I was able to control my Gopro 9 camera. I was able to take a single shot. I'm thinking of writing a full program where I only need to connect to BLE one time and have it open for the rest of the time. The program I will be writing is for timelapse. Is it possible to maintain/keep a single Bluetooth connection? This way I don't have to keep on making several connections while trigger a shutter.

Can we use OpenGopro with a USB cable? If so, is there an example code I can try?

**Comment by rupello** [comment]
Yes, you can use USB to connect  - see https://gopro.github.io/OpenGoPro/#usb
